crypto/asn1/charmap.h

```diff
@@ -2,7 +2,7 @@
  * WARNING: do not edit!
  * Generated by crypto/asn1/charmap.pl
  *
- * Copyright 2000-2020 The OpenSSL Project Authors. All Rights Reserved.
+ * Copyright 2000-2021 The OpenSSL Project Authors. All Rights Reserved.
  *
```

Comment:
Do we really need the copyright year change without any other changes?

Reply:
Yes, unfortunately we do need to manually update the copyright year, due to an issue I've reported earlier today: #13765.
Yet this won't be part of this PR any more as soon as I've rebased this to the (same) workaround meanwhile is merged.